Task3/Task3_Atanas_Kisyov/src/python/main.py

```diff
@@ -0,0 +1,118 @@
+import logging
+
+import argparse
+import psycopg2
+
+
+AVAILABLE_TABLES = (
+    'car_washes',
+    'places',
+    'car_wash_places',
+    'services',
+    'car_wash_services',
+    'users',
+    'accounts',
+    'orders',
+    'order_details',
+)
+INSTRUCTION_MESSAGE = 'This is a list of all available tables in the database. ' \
```

Comment:
no. Please remove this. Constant is variable with config or some general value. In your case, it is a help message.